WMF Framework/EventLoggingService.swift

```diff
@@ -9,8 +9,6 @@ enum EventLoggingError {
 @objc(WMFEventLoggingService)
 public class EventLoggingService : NSObject, URLSessionDelegate {
```

Comment:
You'll want to migrate the old values over to the new UserDefaults so we don't lose these settings when they upgrade to the new version. I would add a new method here, `@objc func migrateShareUsageAndInstallIDToUserDefaults()` where you look up the old value with `libraryValue(for: Key.isEnabled)` and set it in UserDefaults. Then to perform the migration in `MWKDataStore.m`:

1. Set the `WMFCurrentLibraryVersion` at the top to `12`:
```
static const NSInteger WMFCurrentLibraryVersion = 12;
```

2. Add a migration chunk of code at the end of `performUpdatesFromLibraryVersion`:

```
if (currentLibraryVersion < 12) {
        [[EventLoggingService sharedInstance] migrateShareUsageAndInstallIDToUserDefaults];
        [moc wmf_setValue:@(12) forKey:WMFLibraryVersionKey];
        if ([moc hasChanges] && ![moc save:&migrationError]) {
            DDLogError(@"Error saving during migration: %@", migrationError);
            return;
        }
    }
```

You can test by toggling sharing on from `main`, note the installID assigned. Then launch again from this branch and make sure the toggle is still on and the installID hasn't changed.

Reply:
Done, and tested as you suggested.  It seems to be working well.